tox.ini

```diff
@@ -24,6 +27,8 @@ deps =
     # as a dependency of other dependencies and we have to limit
     # the version for compatibility with Numpy.
     oldestdeps: matplotlib==3.0.*
+    devdeps: :NIGHTLY:numpy
+    devdeps: git+https://github.com/astropy/astropy.git#egg=astropy
```

Comment:
I tried to use the astropy dev wheels from Azure here but it didn't work. It keeps reporting conflicts.